Assets/Scripts/Model/AutoFabric.cs

```diff
@@ -5,13 +5,13 @@ namespace Management
 {
     public class AutoFabric : Fabric
     {
-        public new static int StartBuildTime { get; } = 7;
-        public new static int MaxMat { get; } = 2;
-        public new static int BuildPrice { get; } = 10000;
-        protected new static int[] _proc_price { get; } = { 0, 2000, 3000 };
+        override public int StartBuildTime { get; } = 7;
+        override public int MaxMat { get; } = 2;
+        override public int BuildPrice { get; } = 10000;
+        override protected int[] _proc_price { get; } = { 0, 2000, 3000 };
```

Comment:
It's also an ambiguous abbreviation.

Disclaimer: all abbreviations are ambiguous except the really common ones like "cnt", "const" etc.

I would name it `_process_prices` (using "s" at the end because it's plural).